micro-ui/web/micro-ui-internals/README.md

```
@@ -62,6 +62,14 @@ To run this project, you will need to add the following environment variables to
 
 [sample .env file](https://github.com/egovernments/Digit-Core/blob/workbench/frontend/micro-ui/web/micro-ui-internals/example/.env-unifieddev)
 
+## Environment Variables
+
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)